exercises/concept/salary-calculator/src/test/java/SalaryCalculatorTest.java

```diff
@@ -16,75 +16,82 @@ public void setUp() {
+    @DisplayName("The multiplierPerDaysSkipped method returns 1.0 when daysSkipped is equal to the threshold")
+    public void multiplierPerDaysSkippedWhenDaysSkippedIs5() {
+        assertThat(calculator.multiplierPerDaysSkipped(5)).isEqualTo(0.85);
     }
 
     @Test
-    @Tag("task:3")
-    @DisplayName("The finalSalary method returns the correct result when daysSkipped above threshold")
-    public void skippedAboveThreshold() {
-        assertThat(calculator.finalSalary(7, 0)).isEqualTo(850.0);
+    @Tag("task:1")
+    @DisplayName("The multiplierPerDaysSkipped method returns 0.85 when daysSkipped is above the threshold")
+    public void multiplierPerDaysSkippedWhenDaysSkippedIs6() {
+        assertThat(calculator.multiplierPerDaysSkipped(6)).isEqualTo(0.85);
     }
-    
+
     @Test
-    @Tag("task:3")
-    @DisplayName("The finalSalary method returns the correct result when productsSold below threshold")
-    public void soldBelowThreshold() {
-        assertThat(calculator.finalSalary(0, 10)).isEqualTo(1100.0);
+    @Tag("task:2")
+    @DisplayName("The multiplierPerProductsSold method returns 10 when productsSold is below the threshold")
+    public void multiplierPerProductsSoldWhenProductsSoldIs19() {
+        assertThat(calculator.multiplierPerProductsSold(19)).isEqualTo(10);
     }
 
     @Test
-    @Tag("task:3")
-    @DisplayName("The finalSalary method returns the correct result when productsSold above threshold")
-    public void soldAboveThreshold() {
-        assertThat(calculator.finalSalary(0, 25)).isEqualTo(1325.0);
+    @Tag("task:2")
+    @DisplayName("The multiplierPerProductsSold method returns 13 when productsSold is equal to the threshold")
+    public void multiplierPerProductsSoldWhenProductsSoldIs20() {
+        assertThat(calculator.multiplierPerProductsSold(20)).isEqualTo(13);
     }
-    
+
     @Test
-    @Tag("task:3")
-    @DisplayName("The finalSalary method returns the correct result when daysSkipped and productsSold below threshold")
-    public void skippedBelowThresholdAndSoldBelowThreshold() {
-        assertThat(calculator.finalSalary(2, 5)).isEqualTo(1050.0);
+    @Tag("task:2")
+    @DisplayName("The multiplierPerProductsSold method returns 13 when productsSold is above the threshold")
+    public void multiplierPerProductsSoldWhenProductsSoldIs21() {
+        assertThat(calculator.multiplierPerProductsSold(21)).isEqualTo(13);
     }
 
     @Test
-    @Tag("task:3")
-    @DisplayName("finalSalary method returns correct result when daysSkipped below and productsSold above threshold")
-    public void skippedBelowThresholdAndSoldAboveThreshold() {
-        assertThat(calculator.finalSalary(4, 40)).isEqualTo(1520.0);
-    }   
-     
+    @Tag("task:2")
+    @DisplayName("The bonusForProductsSold method returns 5 when productsSold is below the threshold")
+    public void bonusForProductsSoldWhenProductsSoldIs5() {
+        assertThat(calculator.bonusForProductsSold(5)).isEqualTo(50);
+    }
+
```

Comment:
I would expect another test case for `bonusForProductsSold` covering the case where `productsSold` is equal to the threshold, similar to the test case for `multiplierPerProductsSold`

Reply:
I think you're right, thanks for clarifying!